app/src/main/java/com/nervousfish/nervousfish/data_objects/SimpleKey.java

```diff
@@ -0,0 +1,58 @@
+package com.nervousfish.nervousfish.data_objects;
+
+/**
+ * Simple variant of {@link IKey}.
+ */
+public final class SimpleKey implements IKey {
+
+    private final static String TYPE = "simple";
```

Comment:
I don't agree on this one, as it is more of an example class. Having said that I will update the class JavaDoc 😉